packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -652,6 +660,13 @@ export const recoveryPhoneRoutes = (
         return recoveryPhoneHandler.exists(request);
       },
     },
+    {
+      method: 'POST',
+      path: '/message-status',
```

Comment:
Seems like this route should be authenticated?

Reply:
It's protected by the twilio account sid. It needs to be public endpoint though that twilio can call. If the proper account sid is not provided, the request is just terminated and is a no op.

Reply:
Oh I meant based on their [doc](https://www.twilio.com/docs/usage/webhooks/webhooks-security#validating-signatures-from-twilio), they recommended verifying the `X-Twilio-Signature` header using their library. You could make a new auth-scheme or do it route handler.